### DISCUSSION TITLE
Download as CSV

### DISCUSSION THREAD

**Discussion by arroy04** [f1db/f1db#27] [Q&A] [answered]
Hello all! I was teaching myself some SQL and wanted to do some queries for fun on this f1 dataset. I was wondering how to download this data as csv files. When I download it, the files are yml. Is this possible? Thank you so much.

**Comment by marceloverdijk** [comment, marked as answer]
Hi @arroy04 ,

Thanks for reaching out!

If you browse to: https://github.com/f1db/f1db/releases, you will see that every release has various assets:

<img width="905" alt="image" src="https://github.com/f1db/f1db/assets/197004/a8025430-ac29-4e2c-a354-66eb45b5309e">

Just download the [f1db-csv-2024.0.0.beta3.zip](https://github.com/f1db/f1db/releases/download/v2024.0.0.beta3/f1db-csv-2024.0.0.beta3.zip) zip file which contains the csv files.

If this solves your problem please mark the question as answered.

And of course enjoy teaching yourself SQL 💪

**Reply by arroy04** [reply]
Thank you so much!

**Reply by Hayessc2** [reply]
Hello,

Thank you for sharing the link. I was facing the same issue. Is there a way to navigate to this page from the main https://github.com/f1db/f1db link or do I need to bookmark the https://github.com/f1db/f1db/releases page?

**Reply by marceloverdijk** [reply]
I see what you mean now.
Github shows automatically a link to the releases on the right side of the webpage, but I have added some badges now as well that link to the releases. Hope that makes it easier to navigate.

![image](https://github.com/f1db/f1db/assets/197004/cea5af0a-f058-4368-bb64-9927e104e9ba)

And now it's time to watch the first race of the season ;-)

**Reply by Hayessc2** [reply]
Thank you! This is extremely helpful! 
Hope you enjoyed the race!
